.github/workflows/download-link-check-issue.yml

```diff
@@ -0,0 +1,94 @@
+name: Scheduled DVC Download Link Check
+
+on:
+  workflow_call:
+    inputs:
+      lychee_args:
+        description: The args to provide to lychee-action
+        required: true
+        type: string
+      owner:
+        description:
+          The owner of repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.owner.login }}'
+      repo:
+        description:
+          The repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.name }}'
+      label:
+        description:
+          The label that will be used to identify existing link-check reports to
+          edit
+        required: false
+        type: string
+        default: link-checker
+      labels:
+        description:
+          The labels that will be assigned to issues created by this workflow
+        required: false
+        type: string
+        default: link-checker
+      title:
+        description: The title of the issue created by this workflow
+        required: false
+        type: string
+        default: 'Link Checker Report'
+jobs:
+  check_links_and_make_issue:
+    name: Check Download Links and Raise Issue
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Lychee
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.0.8
+        with:
+          args: ${{ inputs.lychee_args }}
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Find latest open Link Check issue
+        uses: actions/github-script@v6
+        id: find-existing-comment
+        env:
+          OWNER: ${{ inputs.owner }}
+          REPO: ${{ inputs.repo }}
+          LABEL: ${{ inputs.label }}
+        with:
+          script: |
+            const { OWNER, REPO, LABEL } = process.env
+            const options = {
+              owner: OWNER,
+              repo: REPO,
+              labels: LABEL,
+              state: "open",
+              creator: "github-actions[bot]",
+              sort: "created"
+            }
+            const issues = await github.rest.issues.listForRepo(options)
+            if(issues && issues.data && issues.data[0]) {
+              return issues.data[0].number
+            }
+
+      - name: Create or Update Issue
+        uses: peter-evans/create-issue-from-file@v4
+        if: steps.lychee.outputs.exit_code != 0
+        with:
+          issue-number: ${{ steps.find-existing-comment.outputs.result }}
+          title: ${{ inputs.title }}
+          content-filepath: ./lychee/out.md
+          labels: ${{ inputs.labels }}
+
+      - name: Close if Check Passes
+        uses: peter-evans/close-issue@v2.0.0
+        if: >-
+          steps.lychee.outputs.exit_code == 0 &&
+          steps.find-existing-comment.outputs.result
```

Comment:
Only run if the lychee run succeeds and an open link check report comment is found.